src/pools/gyroEV2Pool/gyroEV2Math/gyroEV2MathHelpers.ts

```diff
@@ -0,0 +1,15 @@
+import { BigNumber, parseFixed } from '@ethersproject/bignumber';
+import { WeiPerEther as ONE } from '@ethersproject/constants';
+import { mulDown } from '../../gyroHelpers/gyroSignedFixedPoint';
+
+export function normalizeBalances(
```

Comment:
There's already a helper for this is `src/pools/gyroHelpers/helpers.ts` that can be reused.

Reply:
Hey @johngrantuk, this one is slightly different because it multiplies by `tokenRates[index]`